common/djangoapps/student/tests/test_credit.py

```diff
@@ -86,8 +86,8 @@ def setUp(self):
     def test_not_eligible_for_credit(self):
         # The user is not yet eligible for credit, so no additional information should be displayed on the dashboard.
         response = self._load_dashboard()
-        self.assertNotContains(response, "credit-eligibility-msg")
```

Comment:
Do NOT touch credit. We *just* fixed it, and I am 99% certain you broke it with these changes.